service/src/main/kotlin/io/provenance/explorer/service/async/AsyncCaching.kt

```diff
@@ -111,7 +111,7 @@ class AsyncCaching(
     private fun calculateBlockTxFee(result: List<Abci.TxResponse>, height: Int) = transaction {
         result.map { it.tx.unpack(TxOuterClass.Tx::class.java) }
             .let { list ->
-            val numerator = list.sumBy { it.authInfo.fee.amountList.first().amount.toInt() }
+            val numerator = list.sumBy { it.authInfo.fee.amountList.firstOrNull()?.amount?.toInt() ?: 0 }
```

Comment:
nice

Reply:
cuz sometimes people forget that stuff, and then my shit blows up.